my_agent/crash_agent.py

```diff
@@ -0,0 +1,52 @@
+from typing import TypedDict, Annotated, Sequence
+
+from langchain_core.messages import AIMessage
+from langchain_core.runnables import RunnableConfig
+from langgraph.checkpoint.memory import MemorySaver
+from langgraph.graph import END, StateGraph
+from langchain_core.messages import BaseMessage
+from pydantic import BaseModel
+
+from pydantic import ValidationError
+from langgraph.graph import add_messages
+
+
+class CrashExampleConfig(BaseModel):
+    task_id: str = "2997$10071"
+    model_name: str = "claude-3-7-sonnet@20250219"
+
+class CrashExampleState(TypedDict, total=False):
+    task_id: str
+    messages: Annotated[Sequence[BaseMessage], add_messages]
+
+
+def start_task(
+    state: CrashExampleState,
+    config: RunnableConfig,
+) -> CrashExampleState:
+    try:
+        conf = CrashExampleConfig.from_runnable_config(config)
```

Comment:
1. `CrashExampleConfig` is a Pydantic `BaseModel`.
2. `BaseModel` does not have method `from_runnable_config()`.
3. Therefore, an `AttributeError` is raised (see screenshot).

I don't know where `from_runnable_config()` comes from.